.github/workflows/check-deno.yml

```diff
@@ -0,0 +1,28 @@
+name: Deno
+
+on:
```

Comment:
The script doesn't currently support it, but would it make sense to run this on push to? (that is, if my other comment doesn't make sense,